tests/integration/components/frost-fixed-table-test.js

```diff
@@ -327,4 +344,154 @@ describe(test.label, function () {
+          columns=fixedColumns
+          hook=myHook
+          items=heroes
+        }}
+      `)
+
+      return wait()
+    })
+
+    describe('the header', function () {
+      const headerRow = -1  // differentiates from data row 0
+
+      describe('the left section', function () {
+        describe('when an event is triggered by a renderer', function () {
+          beforeEach(function () {
+            // FIXME: Fix this to use qualifiers on '...renderer-input' hook
+            // the renderer has a hook, but the input inside of it doesn't have the right qualifiers.
+            const cellRenderer = $hook('myTable-header-left-cell-renderer', {column: 0})
+            $('input', cellRenderer).val('foo').trigger('input')
+            return wait()
+          })
+
+          it('it should be emitted by the table', function () {
+            expect(dispatcher).to.have.been.calledWith('input', headerRow, 0, 'foo')
+          })
+        })
+      })
+
+      describe('the middle section', function () {
+        const middleColumns = fixedColumnsWithCustomRenderers.slice(1, 3)
+        middleColumns.forEach((column, middleColIndex) => {
+          const colIndex = middleColIndex + 1  // offset by single left column
+          describe(`when the renderer for column ${colIndex} triggers an event`, function () {
+            beforeEach(function () {
+              // FIXME: Fix this to use qualifiers on '...renderer-input' hook
+              const cellRenderer = $hook('myTable-header-middle-cell-renderer', {column: colIndex})
+              $('input', cellRenderer).val('foo').trigger('input')
+              return wait()
+            })
+
+            it('it should be emitted by the table', function () {
+              expect(dispatcher).to.have.been.calledWith('input', headerRow, colIndex, 'foo')
+            })
+          })
+        })
+      })
+
+      describe('the right section', function () {
+        describe('when an event is triggered by a renderer', function () {
+          beforeEach(function () {
+            // FIXME: Fix this to use qualifiers on '...renderer-input' hook
+            const cellRenderer = $hook('myTable-header-right-cell-renderer', {column: 3})
+            $('input', cellRenderer).val('foo').trigger('input')
+            return wait()
+          })
+
+          it('it should be emitted by the table (with global column index 3)', function () {
+            expect(dispatcher).to.have.been.calledWith('input', headerRow, 3, 'foo')
+          })
+        })
+      })
+    })
+
+    describe('the body', function () {
+      describe('the left section', function () {
+        const leftColumns = fixedColumnsWithCustomRenderers.slice(0, 1)
+
+        heroes.forEach((hero, rowIndex) => {
+          leftColumns.forEach((column, columnIndex) => {
+            describe(`when an event is triggered from the cell in
+                      row: ${rowIndex}, column: ${columnIndex}`, function () {
```

Comment:
Here's another example of where I think generating the description passed into `describe` will make this format nicer.